routing/pathfind_test.go

```diff
@@ -616,6 +616,124 @@ func TestKShortestPathFinding(t *testing.T) {
 	assertExpectedPath(t, paths[1], "roasbeef", "satoshi", "luoji")
 }
 
+func TestNewRoute(t *testing.T) {
+
+	var sourceKey [33]byte
+	sourceVertex := Vertex(sourceKey)
+
+	const (
+		startingHeight = 100
+		finalHopCLTV   = 1
+	)
+
+
+	createHop := func(feeRate lnwire.MilliSatoshi, 
+		capacity btcutil.Amount) (*ChannelHop) {
+
+		return &ChannelHop {
+			ChannelEdgePolicy: &channeldb.ChannelEdgePolicy {
+				Node: &channeldb.LightningNode{},
```

Comment:
I think we should set the `baseFee` here for clarity. Either as a parameter, or just `BaseFee: 0`. I think it makes sense as a parameter, as that lets us also test hops with non-zero base fee.

Reply:
Added